src/applications/financial-status-report/components/shared/ReviewPageHeader.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VaButton } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+/**
+ * ReviewPageHeader - A reusable component for rendering "Edit section" headers and buttons on the review page.
+ *
+ * @param {String} [title] - Title of section.
+ * @param {Function} goToPath - goToPath passed in from CustomPage/CustomPageReview props.
+ * @return {React Component}
+ */
+const ReviewPageHeader = ({ title, goToPath }) => {
+  return (
+    <div className="form-review-panel-page" key="income-edit-navigation">
+      <div className="form-review-panel-page-header-row">
+        <h4 className="form-review-panel-page-header vads-u-font-size--h5">
```

Comment:
Can you remove the `vads-u-font-size--h5`? I wanted the section titles to look a little bigger than the individual field titles to differentiate the visual hierarchy.